src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -107,6 +108,25 @@ public MauiApp Build()
 				}
 			});
 
+			_hostBuilder.ConfigureAppConfiguration((hostingContext, config) =>
+			{
+				var env = hostingContext.HostingEnvironment;
+
+				config.AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
```

Comment:
I think some of this is problematic on certain systems. For example, I think file watching is problematic outside of Windows and Mac. And on Android, reading files from the file system at startup is allegedly a huge perf hit. That's why we removed it until we can verify what cost it has on startup time.

Reply:
But on my project I’m using it. 

https://github.com/hflexgrig/MAUI---CleanArchitecture

But it is embedded content and reading as stream from an assembly, maybe that’s why I didn’t see any startup issues. Ok on my project I’ll change json file properties into content and copy if newer, and will measure with timer and let you know the results.

Reply:
@Eilon Btw can it be used as embedded resource and added to default templates when creating Maui app? So it will be different from web apps, where it is content

Reply:
https://github.com/hflexgrig/MAUI---CleanArchitecture/blob/main/MAUI.CleanArchitecture/MauiProgram.cs

Reply:
For mobile isn't a good practice to read a File during startup, if this file gets bigger that will impact even more the startup time. Maybe you can't see a huge impact in a sample app, but I'm sure that you can see it in a real-world application, where you've a lot of features and services

Reply:
@pictos ok, what about embedded resources and stream? Should I change the to that way ? Because having cones as json would be great, I don’t like dictionary staff as it is now.

Reply:
@hflexgrig that could work very well on desktop apps, but not for mobile. Most people, across the world, have low-end devices, and having to load/read a File into memory during startup could not be ideal. But let's what the team thinks about it.

Reply:
@pictos but if we change content into embedded resource, it becomes like in memory collection, and shouldn’t be any performance issues.